config/application.yml.default

```diff
@@ -168,6 +168,7 @@ reset_password_email_window_in_minutes: '60'
 risc_notifications_local_enabled: 'false'
 risc_notifications_eventbridge_enabled: 'false'
 ruby_workers_enabled: 'true'
+rules_of_use_updated_at: '2021-07-06T00:00:00Z'
```

Comment:
The last time we updated this was https://github.com/18f/identity-site/pull/630

So I think the value should be

```suggestion
rules_of_use_updated_at: '2021-05-21T00:00:00Z'
```

And then next time we update the value, we can set it directly in prod's S3 yml